include/psabpf.h

```diff
@@ -32,15 +32,19 @@ typedef struct psabpf_btf {
     /* BTF metadata are associated with eBPF program, eBPF map may do not own BTF */
     int associated_prog;
     void * btf;
+    /* To create map (tuples) with BTF info */
+    int btf_fd;
 } psabpf_btf_t;
 
 typedef struct psabpf_bpf_map_descriptor {
     int fd;
     uint32_t type;
     uint32_t key_size;
     uint32_t value_size;
-    uint32_t btf_type_id;  // TODO: key type ID and value type ID
     uint32_t max_entries;
+    uint32_t btf_type_id;  // TODO: remove, use instead key_type_id and value_type_id
```

Comment:
Yes, it is still needed, removing it require code refactor, because in a lot of places code starts searching right type from `btf_type_id` for map key/value. (I've found ~25 matches for `btf_type_id`)